ui/src/Main.elm

```diff
@@ -598,26 +581,30 @@ tickLabel =
 tickTime : Tick.Time -> Tick.Config msg
 tickTime time =
     let
+        -- interval =
+        --     time.interval
+        -- month =
+        --     { interval | unit = Tick.Month }
         label =
             Junk.label Color.black (Tick.format time)
     in
     Tick.custom
-        { position = 6
+        { position = toFloat (Time.posixToMillis time.timestamp)
         , color = Color.black
         , width = 1
         , length = 7
         , grid = True
-        , direction = Tick.positive
+        , direction = Tick.negative
         , label = Just label
         }
 
 
-xAxisConfig : Axis.Config Info msg
+xAxisConfig : Axis.Config EvictionHistory msg
 xAxisConfig =
     Axis.custom
         { title = Title.default "Month"
         , variable = Just << .date
-        , pixels = 700
+        , pixels = 1000
```

Comment:
this will likely change again when we do a pass trying to fit everything to mobile